docker/images/node/Dockerfile

```diff
@@ -15,18 +12,15 @@ RUN apt-get update \
     && apt-get -y --purge autoremove \
     && rm -rf /var/lib/apt/lists/*
 
-COPY package.json package.json
-COPY gulpfile.babel.js gulpfile.babel.js
-COPY front/ front
-
 # we use `--no-optional` because some optional npm dependencies fail on install
 RUN npm install -g gulp-cli
 # Took decision from SO: https://goo.gl/3uXZxw
 RUN npm install imagemin-jpegtran imagemin-svgo imagemin-gifsicle imagemin-optipng --save
 RUN npm install --no-optional
-RUN gulp build
 
-# @todo #236:30m Add CMD command to nodejs/Dockerfile.
-#  CMD must call "gulp watch" or "gulp build" depending on the presence of
-#  frontend built files. A  command example:
-#  bash -c "[ '$(ls -A /usr/app/src_front/front)' ] && gulp watch || gulp build"
+COPY package.json package.json
+COPY gulpfile.babel.js gulpfile.babel.js
+COPY front/ front
+
+# Keeps track of changes if static files are built, otherwise builds them
+CMD bash -c "[ '$(ls -A /usr/app/src/build)' ] && gulp watch || (gulp build && gulp watch)"
```

Comment:
seems it's "safely watch" function. Looks cool.
Let's update Makefile's rule like in this way:
```
watch-static:
	$(dc) run --rm nodejs
```